README.md

```diff
@@ -54,20 +54,15 @@ rm empty-settings.xml
 ### Update rule description
 
 ```shell
-cd sonarpedia-secrets
-java -jar ../../sonar-rule-api/target/rule-api-2.4.0-SNAPSHOT.jar update
-cd ../sonarpedia-text
-java -jar ../../sonar-rule-api/target/rule-api-2.4.0-SNAPSHOT.jar update
+mvn exec:exec@update --non-recursive -Penable-rule-api -Drules-metadata.directory=sonarpedia-secrets
+mvn exec:exec@update --non-recursive -Penable-rule-api -Drules-metadata.directory=sonarpedia-text
 ```
 
 ### Generate files to include new secrets
 
 After the change, addition or removal of secret specifications, this script can be run to generate the Java classes that are needed 
 for the inclusion or deletion of these secrets.
 
-To run this script you should specify the file name of the rule-api jar `<ruleApiFileName>`, otherwise a default value of `rule-api-snap.jar` will be assumed.
-The rule-api jar needs to be located in the root folder of the project.
-
```

Comment:
Please include, that this script also takes care of generating static rspec files for new rules automatically.